features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Description
+
+This document describes how to update the mbed OS LWIP stack. The mbed OS LWIP stack is a copy of the LWIP master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
+
+When you add new releases or single commits from the LWIP master repository, you must fetch them using the mbed OS LWIP fork repository. Use the repository to rename LWIP source files with the `lwip_` prefix to make them compatible with the mbed OS build system.
+
+* The LWIP master repository is part of the [LWIP project](https://savannah.nongnu.org/projects/lwip).
+* The mbed OS LWIP fork repository is part of [ARMmbed](https://github.com/ARMmbed/lwip).
```

Comment:
- Please change "The mbed OS" to "The Mbed OS".
- Please put "ARMmbed" in code format.